### DISCUSSION TITLE
My website

### DISCUSSION THREAD

**Discussion by Harold-Anderson** [arthelokyo/astrowind#478] [Show and tell]
Here is my website made with Astrowind:  [ACFO website](https://acfo.co) .  Thanks for the template!  The repository is [here](https://github.com/Harold-Anderson/acfo)

**Comment by Sloopdog** [comment]
Your website looks great! and super fast. Good job! I'm currently building mine as well. for photography.  Thank you so much for posting your repo I was having trouble with getting images in my blog post to work and your files help me figure it out. thank you so much! I litterly spent 2 days trying to figure out how to use local optimize images instead of web hosted images. I am not proud with how long it took me to figure that out lol

**Comment by Harold-Anderson** [comment]
Thanks!  I think I worked on that website for two months, so don't feel bad.   I spent a super-long time trying to get _BlogHighlightedPosts_ to work.  It is broken in the template, but if you root through my repo, you'll see that I fixed it.  The official Astrowind template sorts by date.  I changed the sorting to sort by the post filename, so posts can be displayed in the desired order.   _BlogHighlightedPosts_ was supposed to work differently, with a postID[] array, but it was undocumented and didn't work anyway.  Now it displays the top posts, and I've used it in [my website](https://acfo.co), which the official template couldn't do because it was broken.
